decorator.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
```

Comment:
- [ ] I think it will be better if this nameable attribute can be modified outside of the class, so I will suggest that you add an attribute accessor to it 👍🏽 

e.g

`attr_accessor :nameable`